src/main/resources/application.yml

```diff
@@ -75,4 +76,8 @@ management:
 
 graylog:
   host: ${IEXEC_CORE_GRAYLOG_HOST:localhost}
-  port: ${IEXEC_CORE_GRAYLOG_PORT:12201}
\ No newline at end of file
+  port: ${IEXEC_CORE_GRAYLOG_PORT:12201}
+
+stdout:
```

Comment:
I think we should keep days here & use `TimeUnit.DAYS.toMillis(purgeRate)`.

Reply:
Done